pkg/expapi/types.go

```diff
@@ -361,3 +361,34 @@ type ThirdPartyResourceDataList struct {
 	// Items is a list of third party objects
 	Items []ThirdPartyResourceData `json:"items"`
 }
+
+// LimitCapability defines the privilege and capability limits of PodSpecs in a namespace
```

Comment:
Is there a plan to support the concept of `users` in addition to namespaces in the future? If so, would we want to apply these capabilities on users or an entire namespace?